src/Python/Annotation/MultiLineText.md

```diff
@@ -0,0 +1,4 @@
+This example demonstrates the use of multiline 2D text using vtkTextMapper.
+It shows several justifications as well as single-line and multiple-line text inputs.
+
+Author: Pavan Kumar B N 
```

Comment:
here is no need for this line, since git already knows who you are. So please delete it.